packages/devtools_app/test/inspector/inspector_service_test.dart

```diff
@@ -412,13 +417,16 @@ void main() async {
           FlutterTreeType.renderObject,
           isSummaryTree: false,
         ))!;
-        expect(
-          treeToDebugString(selection),
-          equalsIgnoringHashCodes(
-            'RenderParagraph#00000 relayoutBoundary=up2\n'
-            ' └─text: TextSpan\n',
-          ),
-        );
+        // ignore: dead_code
+        if (!ignoreGoldensForNow) {
```

Comment:
we shouldn't need to ignore this part. The part of the txt goldens that flake are lists of items that are not sorted the same way everytime, so this should be fine.

Reply:
It fails with:
```
Expected: multi line description equals RenderParagraph#00000 relayoutBoundary=up2
           └─text: TextSpan
          
  Actual: 'RenderParagraph#7131e relayoutBoundary=up3\n'
            ' └─text: TextSpan\n'
            ''
   Which: expected normalized value
            'RenderParagraph#00000 relayoutBoundary=up2\n'
            ' └─text: TextSpan\n'
            ''
          but got
            'RenderParagraph#00000 relayoutBoundary=up3\n'
            ' └─text: TextSpan\n'
            ''
```
Do you know what that might be about?

Reply:
Is this only failing on master? something may have changed in the framework. For the other golden checks, we use `equalsGoldenIgnoringHashCodes`, which checks the `SHOULD_TEST_GOLDENS` environment variable in its `matches` method. This environment variable is only set to `true` for the bots where we use a pinned version of Flutter. 

We should create another helper method like `equalsGoldenIgnoringHashCodes` that takes a value instead of a file path, and checks `SHOULD_TEST_GOLDENS` before comparing. We have an existing matcher `_AlwaysTrueMatcher` that we can return if `SHOULD_TEST_GOLDENS` is false.